
angle_aware:
  sigma: 0.1
  delta_decrease: 5
  gamma: 0.002 ### 互換性のために必要. 一切参照しない
  alpha: 1
  phi:
    range: [[-0.05, 0.05], [-0.05, 0.05], [-0.1,0.1],  [ -1.5707963, -1.0471975511965976], [-3.14159265358979,3.14159265358979]] #xyzvh, v:[-pi/2, -pi/3,]
    density: [0.05, 0.05, 0.05, 0.10471975511965977, 0.2617993878]  #v: pi/30, h: pi/12
    r: 0.05
  threshold_rate: 0.01
  observe_time: 10
  u_max: 0.1
  ref_z: 1
  slack_cost: 1000000000

  
#   psi:
#     range: [[0.1, 1.9], [0.1, 2.9]]
#     density: [0.05, 0.05]
#     npy_name: angle_aware_psi.npy
# output_psi_topic: angle_aware_psi
# output_J_topic: angle_aware_J